tasks/task05/user_factory.py

```diff
@@ -0,0 +1,73 @@
+#Task 05: User factory
+def create_user(*arg, **kwarg):
```

Comment:
here you expect the function to accept well defined non-keyword arguments:
1. `name`
2. `surname`
3. `age`
you can define them as-is here:
```python
def create_user(name, surname, age=42, **kwargs)
```

Reply:
and the python will automatically verify that `name` and `surnamed` arguments are passed, and then there is no need in extra `if len(args) == 2`...

so, we do not need to use `*args` here. real life use case of `*args` in Python is the standard `max` function:
`max` accepts any number of arguments and returns the maximum of them:
```python
m = max(value1, value2, value3, value4)
```